File: govind.cc
#include<bits/stdc++.h> 
   
using namespace std; 
  
int Gov(int a) 
{ 
    int num = 0; 
     
    while(a > 0 || num >= 9) 
    { 
        if(a == 0) 
        { 
            a = num; 
            num = 0; 
        } 
        num += a % 10; 
        a /= 10; 
    } 
    return num; 
} 
  
int main() 
{ 
    int a; 
    cout<<"\n\nEnter the number:\n\n";
    cin>>a;
    cout <<endl<<endl<< Gov(a)<<endl<<endl; 
    return 0; 
} 

File: govind.py
def gov(a):
	sum = 0

	while(a > 0 or sum > 9):

		if(a == 0):
			a = sum
			sum = 0

		sum += n % 10
		a /= 10
    return sum
    # print(sum)


n = 1234
# print ()
gov(n)
